requirements.txt

```diff
@@ -1,3 +1,6 @@
+click==7.1.2
+pandas==1.0.4
+nltk==3.5
 scikit-learn==0.23.1
 # FIXME: Locking werkzeug until the resolution of pallets/werkzeug#1714
-werkzeug==0.16.1
+werkzeug==1.0.1
```

Comment:
i should have done this, but let's get rid of that `#FIXME` comment as the issue has been fixed by the upgrade